### ISSUE TITLE
overwriteFilesGlob does not work when src uses variables and updateFiles

### ISSUE THREAD

**Issue by nsgundy** [TypeStrong/grunt-ts#303]
grunt-ts 5.1.0

When the `overwriteFilesGlob` option is set to `true` and the `src` option uses a variable itself, then `updateFiles` wont't work correctly
Gruntfile.js:
```js
ts: {
      default: {
          tsconfig: {
            tsconfig: 'tsconfig.json',
            ignoreFiles: true,
            updateFiles: true,
            overwriteFilesGlob: true
          },
          src: ['<%= yeoman.app %>/scripts/**/*.ts', 'typings/tsd.d.ts']
      }
}
```
What ends up being written to `tsconfig.json`:
```json
    "filesGlob": [
        "<%= yeoman.app %>/scripts/**/*.ts",
        "typings/tsd.d.ts"
    ],
    "files": [
        "typings/tsd.d.ts"
    ]
```
None of the 33 files that `<%= yeoman.app %>/scripts/**/*.ts` resolves are being written to `tsconfig.json`.
I believe `overwriteFilesGlob` should resolve any variables first before writing it to `filesGlob`.

**Comment by nycdotnet** [comment]
Hi - sorry for the delay.  I've been very busy with a new job and baby and a few other projects.  I will try to look into this as soon as possible.

**Comment by nycdotnet** [comment]
This is resolved via a warning in grunt-ts v5.3.0 which was just pushed to npm. Please comment or re-open if there are any remaining issues. Thank you for contributing! You're in the patch notes.
